pelicun/tools/DL_calculation.py

```diff
@@ -283,7 +283,7 @@ def log_msg(msg, color_codes=None):
 #       HAZUS Earthquake - Buildings and - Transportation
 # TODO: Loss map for HAZUS EQ Transportation
 
-damage_processes = {
+damage_process_approaches = {
```

Comment:
We use the term "dmg_process" (which means "damage process") for an argument of the `calculate` method in our `DamageModel` class, so I wanted to avoid confusion.

Reply:
Each `dmg_process` used in `calculate` is one damage process, while the variable in question here holds several damage processes, right?. I see that a simple singular/plural form might not be clear enough of a difference to avoid confusion. How about `default_damage_processes` or something similar?